src/util/CurrencyInfoHelpers.ts

```diff
@@ -82,6 +82,26 @@ export const getTokenIdForced = (account: EdgeAccount, pluginId: string, currenc
   return tokenId
 }
 
+export function getCurrencyCodeMultiplier(currencyConfig: EdgeCurrencyConfig, currencyCode: string): string {
+  const { currencyInfo, allTokens } = currencyConfig
+  for (const denomination of currencyInfo.denominations) {
+    if (denomination.name === currencyCode) {
+      return denomination.multiplier
+    }
+  }
+
+  for (const tokenId of Object.keys(allTokens)) {
+    const token = allTokens[tokenId]
+    for (const denomination of token.denominations) {
+      if (denomination.name === currencyCode) {
+        return denomination.multiplier
+      }
+    }
+  }
+
+  return '1'
```

Comment:
Silently failing is a bit scary for a function like this. I get why to do it (avoid dealing with nullish cases), but by being lazy, I worry about a bad bug that communicates the wrong amounts to a user, or worse (sends the wrong user input amount to an API).

Reply:
Might I suggest we implement `nativeToDenomination` and `denominationToNative` in the GUI which will wrap this function with the null-checks necessary to be safe? What did these function do in the core when an invalid currencyCode was given? If they silently failed by not converting, then I guess we're not _increasing_ bug surface area, but if they threw an error, then we should perhaps consider some exceptions in the GUI as well.

Reply:
This is an exact copy-paste of the core logic, including the silent failure case. See https://github.com/EdgeApp/edge-core-js/blob/25fc3744fa1275b9ebabc75e3372f05926244e03/src/core/currency/currency-selectors.ts#L8 My goal was to make this a straight move, and not to introduce any new behavior.

The *correct* solution is to make everything based on tokenId's, and not currency codes, and so the new `nativeToDenomination` and `denominationToNative` need to do that, along with the appropriate error handling. However, that will be a follow-on task at best.

Reply:
Same implementation is okay by me.